packages/sui-test/README.md

```diff
@@ -132,6 +132,69 @@ Cypress can be detected as a robot if your server has that kind of protection or
 
 If defined, any error on your tests will create a screenshot of that moment in the `./.tmp/test-e2e/screenshots` folder of your project.
 
+
+# Tools
+
+## Descriptor by environment patcher
+The descriptor by environment is a patch with the purpose of add some extra functionality to our mocha describe and it methods.
+
+### How to import it?
+
+First of all, the patcher MUST BE APPLIED on each test that we want to have the extra methods so at the top of `ourExampleSpec.js` we will add the next code:
+
+```javascript
+import { descriptorsByEnvironmentPatcher } from '@s-ui/test/lib/descriptor-environment-patcher'
+descriptorsByEnvironmentPatcher()
```

Comment:
That is too verbose.

```
import patcher from '@s-sui/test/lib/patcher'
patcher()
```
Is a little less verbose 

The important here is use it in this case like a default naming. And I know that we are saying to avoid it. But for me in this scenario make sense.

Reply:
As you wish, for me is not a bad thing to have this patcher with a verbose and descriptive name as is something that is modifying the base function set of an external library if is not a really ugly think I would like to remain the import in that way to avoid confusions and questions about 'what is this patcher doing'

Reply:
To me, even a register like  `import ''@s-sui/test/lib/describe-env-register"` would be enough.

Reply:
Yup I've started to do as just a import but the thing is that we need that this logic being executed on EACH import. Why? because each the test describe, it etc return a function that can't be prototype modified :/

I've talked with @carlosvillu about that and it was the best approach that we found.

Reply:
If you are really strong with this opinion I don't have any problem to change it but I would like to know and understand the concerns surrounding the default import in this particular case

Reply:
I get it. Then keep it simple. We could prototype extensions like they do in chai by it'll be ugly.